test/appengine/express-memcached-session.test.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var assert = require('assert');
+var request = require('supertest');
+var app = require('../../appengine/express-memcached-session/server.js');
+
+describe('express-memcached-session', function () {
+  it('should return 200 and the correct response', function (done) {
+    request(app)
+      .get('/')
+      .expect(200)
+      .expect(function (res) {
+        assert(res.text.match(/<div>(\d|\d\d|\d\d\d)\.(\d|\d\d|\d\d\d)\.(\d|\d\d|\d\d\d)\.(\d|\d\d|\d\d\d)<\/div>Viewed <strong>\d<\/strong> times./), // jshint ignore:line
```

Comment:
Regexs are like violence, if they aren't working, you aren't using enough of it.